stream-projects/mogul-menu/components/menu/extension-icon/extension-icon.tsx

```diff
@@ -1,36 +1,66 @@
 import {
-  classKebab,
-  getSrcByImageObj,
-  React,
-  Ripple,
-  useStyleSheet,
+    classKebab,
+    getSrcByImageObj, jumper,
+    React,
+    Ripple, useEffect, useState,
+    useStyleSheet,
 } from "../../../deps.ts";
 import { getHasNotification, useTabs } from "../../tabs/mod.ts";
-import { getMenuIconImageObj, useMenu } from "../mod.ts";
+import { getMenuIconImageObj, useMenu, getIsOpen } from "../mod.ts";
 import stylesheet from "./extension-icon.scss.js";
+import hamburgerIcon from "../../../assets/hamburgerIcon.png";
```

Comment:
this won't actually work in prod. vite (what we use in truffle-cli dev) can handle this, but in prod in needs to be pure unbundled js

what we usually do for icons is use the Icon component passing in the icon path https://github.com/trufflehq/truffle-packages/blob/master/stream-projects/patreon/components/premium-content-embed/premium-content-embed.tsx#L120

you should be able to get the icon path by exporting as svg in figma, then viewing source of icon and grabbing the path (assuming the icon has 24x24 viewbox, which rachel typically does - let her know if she didn't)